services/web-app/pages/common-mdx-errors.mdx

```diff
@@ -119,7 +119,7 @@ for a comprehensive list of common MDX parsing errors
 Sample:
 
 <div style={{ backgroundColor: 'lightgray', padding: '30px', borderRadius: '10px' }}>
-  <InlineError
+  <MdxNotification
     title="Component not rendering"
     description="Verify component usage documentation to make sure you're supplying all necessary information in the correct format"
     link="Get support"
```

Comment:
Should this link say "URL documentation" and link to https://platform.carbondesignsystem.com/ ?